docs/reference/intro.mdx

```diff
@@ -10,11 +10,18 @@ OpenFeature is an open standard for feature flagging, created to support a robus
 
 ## What are feature flags?
 
-In the most basic case, you can think of a feature flag as an if/else statement that can be controlled at runtime. Feature flags allow application behavior to be altered without the deployment of new code.
+In the most basic case, you can think of a feature flag as an if/else statement that can be controlled at runtime.
+Feature flags allow application behavior to be altered without the deployment of new code.
 
-This capability serves a variety of purposes. You can reduce the need for long-running feature branches. You can hide work-in-progress functionality from end users, while still exposing it for internal testing. You can perform canary releases - rolling out a new feature to an initially small subset of users. You can perform A/B testing. You can safely degrade parts of a production system that are experiencing an outage. You can restrict access to specific product functionality - an admin page, for example, or premium features that are only exposed to paid subscribers.
+This capability serves a variety of purposes.
+You can reduce the need for long-running feature branches.
+You can hide work-in-progress functionality from end users, while still exposing it for internal testing.
+You can perform canary releases - rolling out a new feature to an initially small subset of users.
+You can perform A/B testing. You can safely degrade parts of a production system that are experiencing an outage.
+You can restrict access to specific product functionality based on user-characteristics such as geography or IP address, for usability, compliance or licensing reasons.
 
```

Comment:
@beeme1mr  this is the only realy change. The rest is just breaking things up to once sentence per line.